<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	text-align: center;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QWidget {
    background-color: rgb(208, 225, 249);
}

QPushButton {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

exp_decrease_pushButton {
	padding: 10;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton:focus {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButton&quot;] {
    background-color: rgb(77, 100, 141);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QPushButton[class=&quot;CustomSpinboxButtonPressed&quot;] {
    background-color: rgb(40, 54, 85);

    color: white;
    border-radius: 5px;
	padding: 25;
	font: 57 18pt &quot;Sono Medium&quot;;
}

QSpinBox {
	text-align: centrel;
    border: none;
	border-radius: 5px;
    background-color: #E6F4FF;
	font: 57 16pt &quot;Sono Medium&quot;;
}

QDoubleSpinBox:focus {
    border: 2px solid #0057D9;
    background-color: #E6F4FF;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QPushButton" name="back_pushButton">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0" columnstretch="0,0,0,0,0,0">
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>-1</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="3">
      <widget class="QSpinBox" name="exp_spinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="iso_decrease_pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="iso_spinBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Экспозиция</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="iso_increase_pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="exp_increase_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="exp_decrease_pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>ISO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="take_photo_camera_1">
     <property name="text">
      <string>Снять фото камерой 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="take_photo_camera_2">
     <property name="text">
      <string>Снять фото камерой 2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
